src/js/utils/option-utils.jsx

```diff
@@ -89,8 +89,8 @@ export const debounceProductsFetch = (waitTime, minSearchLength, locationId) =>
             productCode: obj.productCode,
             handlingIcons: obj.handlingIcons,
             lotAndExpiryControl: obj.lotAndExpiryControl,
-            label: `${obj.productCode} - ${obj.translatedName ?? obj.name}`,
-            hasTranslatedName: !!obj.translatedName,
+            label: `${obj.productCode} - ${obj.displayName ?? obj.name}`,
+            hasDisplayName: !!obj.displayName,
```

Comment:
I am wondering why  `hasDisplayName` has to be a boolean.
What I mean is that we could just pass 
```
displayName: obj.displayName
```
and then we can use this variable the same way we use `hasDisplayName` 
```
if (!!option.deisplayName) {
    ...
}
```
and furthermore we will have access to value of `displayName` if we ever gonna need that.

Reply:
obviously it doesn't have to be. I was also considering your point and called it explicity as boolean, because it was more readable for me at that moment, but I agree it does not really matter and if we wanted to use it somehow in the future, it would be better to make it as `displayName` and do the negations wherever it is needed or even it wouldn't be needed and I could even shorten your `if` to:

```javascript
if (option.displayName) {
 ...
}
```

Thanks for pointing that out, I will change it